pkg/storage/blobstore/s3/s3.go

```diff
@@ -50,22 +52,57 @@ func (s *Storage) Download(path string, w io.WriterAt) error {
 	return nil
 }
 
+func (s *Storage) Delete(path string) error {
+	_, err := s.client.DeleteObject(context.TODO(), &s3.DeleteObjectInput{
+		Bucket: aws.String(s.Bucket),
+		Key:    aws.String(path),
+	})
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
 // NewStorage returns a new initialized Storage
 func NewStorage(c map[string]any) (*Storage, error) {
+	var cfg aws.Config
+	var err error
+	skipDefaultConfig := false
 
+	val, ok := c["skipDefaultConfig"]
```

Comment:
I don't really understand why we need `skipDefaultConfig`.

Reply:
Won't it fail if it doens't found any config from env/context

does this suggested change works?.

![Screenshot from 2024-03-16 23-42-51](https://github.com/scratchdata/scratchdata/assets/47126953/ed7681bc-5432-4921-87cb-dc3ff8e5b023)

Reply:
Perhaps we should replace `LoadDefaultConfig()` with `cfg := aws.Config{}`. We never want to load anything from the environment. We only ever want to load data from this configuration file.

We can create a ticket to be able configuration options from environment variables or credentials files.

Thank you for clarifying.